docs/examples.md

```diff
@@ -417,6 +417,37 @@ gcloud functions deploy image-annotator \
   --entry-point=Google.Cloud.Functions.Examples.StorageImageAnnotator.Function
 ```
 
+## MultiProjectFunction and MultiProjectDependency
+
+All the other examples provided are standalone, but in real world
+projects, often the project containing your function will depend on
+another local project. The [MultiProjectFunction](../examples/Google.Cloud.Functions.Examples.MultiProjectFunction)
+and [MultiProjectDependency](../examples/Google.Cloud.Functions.Examples.MultiProjectDependency)
+directories provide an example of this: the MultiProjectFunction project depends on the
+MultiProjectDependency project using a `<ProjectReference>` MSBuild element.
+
+When deploying a function that depends on another local project, you
+need to ensure that all the relevant source code is uploaded, and
+that you indicate which project contains the function. See the
+[deployment documentation](deployment.md#deploying-a-function-with-a-local-project-dependency)
+for more details.
+
+Sample deployment, from the `examples` directory:
+
+```sh
+gcloud beta functions deploy image-annotator \
```

Comment:
Is the name image-annotator stil meaningful in this example or should this be something like multi-project?

Reply:
Whoops, no - multi-project would indeed be appropriate.